animeal/src/Flows/Main/Modules/Favourites/Favourites/View/FavouriteDetailsViewController.swift

```diff
@@ -0,0 +1,140 @@
+import UIKit
+import UIComponents
+import Style
+
+final class FavouriteDetailsViewController: UIViewController, FeedingPointDetailsViewable {
```

Comment:
Looks like it's unnecessary ViewController.

Reply:
Thanks for catching that. Removed unused VC